feature/bgp/feature.textproto

```diff
@@ -55,6 +72,25 @@ telemetry_path {
 telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/session-state"
 }
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/state/active"
+}
+
```

Comment:
no space here, just to keep consistent with existing style?

Reply:
done!